telegram/ext/commandhandler.py

```diff
@@ -274,15 +290,14 @@ def __init__(self,
                  command,
                  callback,
                  filters=None,
-                 allow_edited=False,
                  pass_args=False,
                  pass_update_queue=False,
                  pass_job_queue=False,
                  pass_user_data=False,
                  pass_chat_data=False):
 
         super(PrefixHandler, self).__init__(
-            'nocommand', callback, filters=filters, allow_edited=allow_edited, pass_args=pass_args,
+            'nocommand', callback, filters=filters, allow_edited=None, pass_args=pass_args,
```

Comment:
Could we change 'nocommand' to None, to make it more clear that it's simply not used?
This of course requires changing a few things in the CommandHandler

Reply:
That would mean changing CommandHandlers logic to accomodate this. This seemed pretty clean to me.